.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,39 @@
 <!--
-Thanks for sending a pull request!
+Thank you for contributing! Please remember to:
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
+- Read our contributing guidelines: https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests
+- Sign your commits with your real name. You can amend previous commits with `git commit -s --amend`.
+-->
+
+#### Description
+<!-- Provide a clear summary of your changes and why they are needed. -->
 
-Finally, be sure to sign commits with your real name. Since by opening
-a PR you already have commits, you can add signatures if needed with
-something like `git commit -s --amend`.
+Fixes:
+<!--
+Link to the issue(s) this Pull Request addresses.
+- If it fixes a single issue, use:
+    Fixes: #00000
+    Fixes: https://github.com/containers/common/issues/00000
+    Fixes: https://issues.redhat.com/browse/RHEL-00000
+    Fixes: RHEL-00000
+- If it fixes multiple issues, use commas:
+    Fixes: #00000, #00001, #0002
+    Fixes: https://github.com/containers/common/issues/00000, https://github.com/containers/common/issues/00001
+    Fixes: https://issues.redhat.com/browse/RHEL-00000, https://issues.redhat.com/browse/RHEL-00001
+    Fixes: RHEL-00000, RHEL-00001
```

Comment:
I don't think that makes sense in terms of the style, one Fixes should only contain one link. If you link multiple they should be on separate lines with Fixes: each. But CONTRIBUTING.md doesn't seem to specify that.

Reply:
I don't think it is particular defined anywhere, but IMO comma separated is less readable especially with long links so I rather promote using one per line